src/OrchardCore/OrchardCore.ContentManagement.GraphQL/OrchardCore.ContentManagement.GraphQL.csproj

```diff
@@ -10,6 +10,8 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Apis.GraphQL.Abstractions\OrchardCore.Apis.GraphQL.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore.Modules\OrchardCore.CustomSettings\OrchardCore.CustomSettings.csproj" />
```

Comment:
You can't reference other modules from a module.

Reply:
Interesting. I thought one could list dependencies in `Manifest.cs`?

So I'll have to move the SiteQuery builder to OrchardCore.Settings and the CustomSettingsSiteGraphBuilder to OrchardCore.CustomSettings with new startups that require the graphql feature, right?